packages/core/integration-tests/test/scope-hoisting.js

```diff
@@ -4625,17 +5267,18 @@ describe('scope hoisting', function() {
       .getBundles()
       .sort((a, b) => b.stats.size - a.stats.size)[0];
     let contents = await outputFS.readFile(sharedBundle.filePath, 'utf8');
-    assert(contents.includes(`$parcel$global[parcelRequireName] = `));
+    assert(contents.includes(`if (parcelRequire == null) {`));
   });
 
-  it('does not include prelude if child bundles are isolated', async function() {
+  it.skip('does not include prelude if child bundles are isolated', async function() {
     let b = await bundle(
       path.join(__dirname, '/integration/worker-shared/index.js'),
     );
 
     let mainBundle = b.getBundles().find(b => b.name === 'index.js');
     let contents = await outputFS.readFile(mainBundle.filePath, 'utf8');
-    assert(!contents.includes(`$parcel$global[parcelRequireName] = `));
+    // We wrap for other reasons now, so this is broken
+    assert(!contents.includes(`if (parcelRequire == null) {`));
   });
```

Comment:
The generated code is broken?

Reply:
The test - the prelude gets included for other reasons